routes/waiter/waiter.controller.js

```diff
@@ -0,0 +1,34 @@
+const Menu = require('./waiter.model');
```

Comment:
Not sure the `/waiter/*` files are needed? seem to be a clone of what is implemented in the `/menu/*` files?